src/factories/nodeFlowRun.ts

```diff
@@ -37,10 +37,19 @@ export async function flowRunContainerFactory(node: RunGraphNode) {
   let hasEvents = false
   let hasArtifacts = false
 
+  let internalNode = node
+  let isOpen = false
+
   container.sortableChildren = true
-  bar.zIndex = 2
-  label.zIndex = 3
-  arrowButton.zIndex = 3
+  const maxContentZIndex = Math.max(
+    DEFAULT_ROOT_EVENT_Z_INDEX,
+    DEFAULT_ROOT_FLOW_STATE_Z_INDEX,
+    DEFAULT_ROOT_ARTIFACT_Z_INDEX,
+  )
+
+  bar.zIndex = maxContentZIndex + 1
+  label.zIndex = maxContentZIndex + 2
+  arrowButton.zIndex = maxContentZIndex + 2
```

Comment:
Shouldn't we just create consts for what we expect these to be rather than making them dynamic?

Reply:
Updated. In my mind I had reserved the definitions in `const` as sort of more global, whereas in this context the z-indexes only apply to this container, but realized that doesn't really matter. More readable just having `subflow` default z-indexes in the const.